src/Aspire.Hosting.Azure.Storage/AzureStorageExtensions.cs

```diff
@@ -49,6 +49,14 @@ public static IResourceBuilder<AzureStorageResource> AddAzureStorage(this IDistr
                 sku: StorageSkuName.StandardGrs
                 );
 
+            if (builder.ExecutionContext.IsPublishMode)
+            {
+                // TODO: Waiting for CDK fix.
+                //// In publish mode we block access to everything except other Azure services.
+                //storageAccount.AssignProperty(p => p.NetworkRuleSet.DefaultAction, "'Deny'");
```

Comment:
How can someone override this, if they want something different?

Reply:
The default value is Allow ... so they can set it back to that. But this is a good question for @tg-msft and @JoshLove-msft around how assignments can be deleted. One of the reasons why the CDK overloads are experimental.